components/Player/PlayerHeader.css

```diff
@@ -82,3 +82,47 @@
     width: 100%;
   }
 }
+
+/*
+  This styles looks like a little hardcoded. I don't think this is a trouble,
+  because postcss already apply encapsulation.
+  So all but the 'global' styles are hardcoded at some kind.
+*/
+.infoRow {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 5px 5px;
+  
+  & > abbr {
+    margin-right: 2px;
+  }
+  
+  & > span:not(:last-child){
+    margin-right: 6px;
+  }
+}
+
+.playerTitle {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+/* no way to override icon size, because it set as inline styles.
+  Actually it's possible to set style property of icon, but i don't know how to
+  get this css class as object in Jsx script.*/
+.playerRegistredIcon {
+  width: 12px !important;
+  height: 12px !important;
+  margin-left: 2px;
+}
+
+.smallCheese {
```

Comment:
can we define the CSS for cheese in one place (palette.css?) and then just resize it for small version?